tapchannel/commitment.go

```diff
@@ -162,6 +165,9 @@ func ComputeView(ourBalance, theirBalance uint64,
 		case lnwallet.Add:
 			continue
 
+		case lnwallet.NoOpAdd:
```

Comment:
Do we also need to replicate the awareness of how this (doesn't) impact the sat balances when evaluated? IIRC, we started to replicate more of that logic here so we derive the correct local/remote balance information.